core/controllers/classifier.py

```diff
@@ -26,6 +31,17 @@
 from core.domain import email_manager
 import feconf
 
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_FUTURE_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'future-0.17.1')
+
+sys.path.insert(0, _FUTURE_PATH)
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+import past.builtins  # isort:skip
```

Comment:
Is this something we're going to need to keep forever ...?

Reply:
If we are not planning to use Python 2 after full migration, then we can replace this with str